src/stories/storybook-helpers.tsx

```diff
@@ -66,9 +69,7 @@ export const InteractiveInput: React.FC<InteractiveInputProps> = ({
   defaultValue,
   ...passedProps
 }) => {
-  const [value, setValue] = useState<string | string[] | number | undefined>(
-    defaultValue
-  );
+  const [value, setValue] = useState<InputValue>(defaultValue);
```

Comment:
Was throwing a type error because `defaultValue` was being seen as readonly when it wasn't explicitly typed.

Reply:
That's funny, I was running into typing issues on `Form`, related to various typed inputs, and I think my hack to get it to pass was no bueno.  I'll try to steal this...